kwave/utils/mapgen.py

```diff
@@ -364,30 +324,25 @@ def water_non_linearity(temp):
     return BonA
 
 
-def make_ball(Nx, Ny, Nz, cx, cy, cz, radius, plot_ball=False, binary=False):
+def make_ball(Nx: int, Ny: int, Nz: int, cx: int, cy: int, cz: int, radius: int, plot_ball: bool = False,
+              binary: bool = False) -> Union[np.ndarray, Tuple[np.ndarray, Any]]:
     """
-    MAKEBALL Create a binary map of a filled ball within a 3D grid.
-
-    DESCRIPTION:
-         make_ball creates a binary map of a filled ball within a
-         three-dimensional grid (the ball position is denoted by 1's in the
-         matrix with 0's elsewhere). A single grid point is taken as the ball
-         centre thus the total diameter of the ball will always be an odd
-         number of grid points.
+    Creates a binary map of a filled ball within a 3D grid.
+
     Args:
-        Nx: size of the 3D grid in x-dimension [grid points]
-        Ny: size of the 3D grid in y-dimension [grid points]
-        Nz: size of the 3D grid in z-dimension [grid points]
-        cx: centre of the ball in x-dimension [grid points]
-        cy: centre of the ball in y-dimension [grid points]
-        cz: centre of the ball in z-dimension [grid points]
-        radius: ball radius [grid points]
-        plot_ball: Boolean controlling whether the ball is plotted using voxelPlot (default = false)
-        binary: Boolean controlling whether the ball map is returned as a double precision matrix (false)
-                or a logical matrix (true) (default = false)
+        Nx: int, size of the 3D grid in x-dimension [grid points].
+        Ny: int, size of the 3D grid in y-dimension [grid points].
+        Nz: int, size of the 3D grid in z-dimension [grid points].
+        cx: int, centre of the ball in x-dimension [grid points].
+        cy: int, centre of the ball in y-dimension [grid points].
+        cz: int, centre of the ball in z-dimension [grid points].
```

Comment:
We should update the args here to be a vector with 3 elements. Maybe add a TODO item?

Reply:
I had done that, but then thought it would confuse the PR if I started changing signatures. Will add TODO